README.rst

```diff
@@ -42,7 +42,7 @@ Example usage for the base model with a template structure in pdb format:
                            --n-samples 16 \
                            --chunk-size 4 \
                            --num-steps=40 \
-                           --cuda \
+                           --cuda-device-index 0 \
```

Comment:
I would gently push back on this as the device index can be controlled by the CUDA_VISIBLE_DEVICES variable. 
Will unblock this PR, but please update readme.md about this option if the code is merged as-is.

Reply:
I can understand the `CUDA_VISIBLE_DEVICES` use case. I personally don't use this approach to selecting GPUs, however, I realize many others do. I'll revert this device index change.

Reply:
Reverted in [this commit](https://github.com/zrqiao/NeuralPLexer/pull/11/commits/9e07a93c31380ab78021450cdce8c620a422400e).